static/templates/reports/pac.hbs

```diff
@@ -5,9 +5,16 @@
       (This report has been amended)
```

Comment:
With the rollback on amendment info on reports, we'll have to keep this in for reports panels.